frontend/src/views/Lobby.tsx

```diff
@@ -639,6 +660,54 @@ function LobbyPage() {
                 />
               </SliderContainer>
             </HoverContainerSlider>
+            <NoMarginMediumText>Room Size</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {size === 31? 'No limit' : (size === 1? '1 person' : `${size} people`)}
+            </NoMarginSubtitleText>
+            <HoverContainerSlider>
+              <HoverElementSlider
+                enabled={isHost(currentUser)}
+                onMouseEnter={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(true);
+                  }
+                }}
+                onMouseLeave={() => {
+                  if (!isHost(currentUser)) {
+                    setHoverVisible(false);
+                  }
+                }}
+              />
+              <SliderContainer>
+                <Slider
+                  min={1}
+                  max={31}
+                  value={size}
+                  disabled={!isHost(currentUser)}
+                  onChange={(e) => {
```

Comment:
nit: This function is pretty large, and might be better suited to go outside as a separate function which this `onChange` action calls.